app/domain/authentication/authn_jwt/identity_from_decoded_token_provider.rb

```diff
@@ -13,7 +13,7 @@ def initialize(authentication_parameters)
       def provide_jwt_identity
         token_field_name = fetch_token_field_name
         @logger.debug(LogMessages::Authentication::AuthnJwt::LOOKING_FOR_IDENTITY_FIELD_NAME.new)
-        jwt_identity = @decoded_token[token_field_name.to_sym]
+        jwt_identity = @decoded_token[token_field_name]
```

Comment:
Do we have UT that was able to figure it out?

Reply:
Or is it a spec file below :-)

Reply:
Yes

Reply:
https://github.com/cyberark/conjur/blob/feature-jwt-authenticator/spec/app/domain/authentication/authn-jwt/identity_from_decoded_token_provider_spec.rb